packages/components/src/test/instance-filter-builder/IntanceFilterBuilder.test.tsx

```diff
@@ -48,6 +49,20 @@ describe("InstanceFilter", () => {
     sinon.restore();
   });
 
+  const testImodel = {} as IModelConnection;
+  const testDescriptor = {} as Descriptor;
+  const testActions = {} as PropertyFilterBuilderActions;
+
+  const rootGroupMock = moq.Mock.ofType<PropertyFilterBuilderRuleGroup>();
```

Comment:
I think this can be changed to simple object as you are not changing behavior of `rootGroupMock` in any of the tests.

Reply:
changed